src/pod.rs

```diff
@@ -1,9 +1,11 @@
 //! Plain data object safe transmute
 //!
-//! Functions in this module are guarded from out-of-bounds memory access as well as from unaligned access,
-//! raising errors on both cases. Moreover, only a safe subset of types (which implement
-//! [`PodTransmutable`](trait.PodTransmutable.html)) can be used as the transmute target, thus ensuring
-//! full safety.
+//! Functions in this module are guarded from out-of-bounds memory access and from unsafe transmute
+//! target types with the use of the [`PodTransmutable`](trait.PodTransmutable.html)) trait.
```

Comment:
"and from unsafe transmutation target types via/through the use of", maybe? Makes more sense to me, the "transmute" threw me off especially.